Checking in modified version of the installer (#498)
grafana/agent: CHANGELOG.md

- [FEATURE] (beta) New integration: windows_exporter (@mattdurham)

> - [FEATURE] (beta) Agent Windows Installer is now built has a release artifact (@mattdurham)

Reviewer: ```suggestion
- [FEATURE] (beta) Grafana Agent Windows Installer is now included as a release artifact. (@mattdurham)
```
